css/partials/_typography.scss

```diff
@@ -42,6 +42,7 @@ a {
 	&.no-underline {
 		&:hover {
 			text-decoration: none;
+			color: #fff;
```

Comment:
This isn't the right place to put this color - it should probably be in _header.scss where the background color is defined as a dark color so we don't end up with white on white.